kagglesdk/kernels/types/kernels_enums.py

```diff
@@ -22,6 +22,12 @@ class KernelsListViewType(enum.Enum):
   RECENTLY_VIEWED = 7
   PUBLIC_AND_USERS_PRIVATE = 8
 
+class KernelVersionType(enum.Enum):
+  KERNEL_VERSION_TYPE_UNSPECIFIED = 0
+  BATCH = 1
+  INTERACTIVE = 2
+  QUICK = 3
+
```

Comment:
I know this is generated, but I think it would be good to clean it up to something more user-friendly.  Can you fix the MT code to generate something that looks like this:

```suggestion
class KernelExecutionType(enum.Enum):
  KERNEL_EXECUTION_TYPE_UNSPECIFIED = 0
  SAVE_AND_RUN_ALL = 1
  INTERACTIVE = 2
  QUICK_SAVE = 3

```

And similarly replace `kernelVersionType` with `kernelExecutionType` throughout (the reason being that interactives aren't versions at all).

Reply:
http://b/418266515